### PR TITLE
Update Verification Pipeline to support v4 images

### PR DESCRIPTION
**Pull request by CooperLink**
### PR information
Updating pipeline verification script to run for both v3 and v4 automated releases.  This PR updates the script, and the github action to work for both v3 and v4 pipelines with no need for another action. 

See v3 Automation PR: 
https://github.com/Azure/azure-functions-docker/pull/500

- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [x] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

**Comment by CooperLink** [comment]
Do not merge until I commit change to AutoPR

File: .github/workflows/autopr.yaml [hunk 1]
```diff
@@ -17,24 +17,23 @@ jobs:
     steps:
     - uses: actions/checkout@v2
       with:
-        ref: dev
+        ref: u/clink/update_verification_v4
```
Comment (pragnagopa) [inline .github/workflows/autopr.yaml]: dev?

Comment (CooperLink) [reply .github/workflows/autopr.yaml]: Just fixed it! Called out in comment above but had a meeting with Bala in between

**Review by balag0 — APPROVED** [review]
Good idea to use the major version parameter 👍

**Comment by TsuyoshiUshio** [comment]
Thank you for the great contribution. 
If the direction will change, could you update this(if not ok)
https://azfunc.visualstudio.com/Azure%20Functions/_releaseDefinition?definitionId=20&_a=definition-tasks&environmentId=215

**Comment by CooperLink** [comment]
@sidkri FYI

**Pull request merged**